ui/server/src/test/scala/pl/touk/nussknacker/ui/api/DefinitionResourcesSpec.scala

```diff
@@ -18,6 +18,21 @@ class DefinitionResourcesSpec extends FunSpec with ScalatestRouteTest with FailF
 
   private implicit final val string: FromEntityUnmarshaller[String] = Unmarshaller.stringUnmarshaller.forContentTypes(ContentTypeRange.*)
 
+  it("should return definition data for allowed classes") {
+    getProcessDefinitionData(existingProcessingType, Map.empty[String, Long].asJson) ~> check {
+      status shouldBe StatusCodes.OK
+
+      val typesInformation = responseAs[Json].hcursor
+        .downField("processDefinition")
+        .downField("typesInformation")
+        .downAt(_.hcursor.downField("clazzName").get[String]("display").right.value == "ReturningTestCaseClass")
```

Comment:
Can you use `classOf[ReturningTestCaseClass].getSimpleName` ? It will be easier to refactor this name in the future and will be easier to navigate how this class is added in test model

Reply:
ReturningTestCaseClass is not visibled in this integrations test package.